lightly/transforms/models/moco_transform.py

```diff
@@ -0,0 +1,81 @@
+from lightly.transforms.models.simclr_transform import SimCLRTransform
+from lightly.transforms.utils import IMAGENET_NORMALIZE
+from typing import Optional, Tuple, Union
+
+
+class MoCoTransform(SimCLRTransform):
+    """Implements the transformations for MoCo v1.
+
+    For MoCo v2, simply use the SimCLR settings.
```

Comment:
I think we can drop the comment about MoCo v2 and just add a "new" `MoCoV2Transform` at the end of the file by setting `MoCoV2Transform = SimCLRTransform` with a comment above that says that moco v2 transform settings are identical to simclr.